From 78884282172fde0828348186df2b32d010b7e105 Mon Sep 17 00:00:00 2001
From: dann frazier <dann.frazier@chainguard.dev>
Date: Tue, 15 Sep 2026 11:33:32 -0600
Subject: [PATCH 09/18] Revert "maint: avoid -Werror=maybe-uninitialized with
 gcc -flto -O3"

This reverts commit 3d3022995b1a981e9e8ed2a5b4347427ff2552d2.
NEWS left untouched.
---
 src/env.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/env.c b/src/env.c
index 8651465fc..dd526b836 100644
--- a/src/env.c
+++ b/src/env.c
@@ -1026,7 +1026,6 @@ main (int argc, char **argv)
   char *argv0 = NULL;
   struct env_vector env_vector;
 #ifdef lint
-  memset (&env_vector, 0, sizeof env_vector);
   char **initial_environ = environ;
 #endif
 
-- 
2.53.0

